webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js

```diff
@@ -6,6 +7,27 @@ import {
 import { translate as __ } from '../../common/I18n';
 import { removeLastSlashFromPath } from '../../common/helpers';
 
+export const checkBrowserSupport = () => {
+  browserUpdate({
+    required: {
+      i: 12, // obsolete IE
+      e: -2,
+      f: -3,
+      o: -3,
+      s: -1,
+      c: -3,
+    },
+    text: __(
```

Comment:
can you please use the translation function only for strings?

Reply:
Done, the text seems like a ungly blob now :disappointed: